docs/reference/connector/apis/update-connector-pipeline-api.asciidoc

```diff
@@ -0,0 +1,101 @@
+[[update-connector-pipeline-api]]
+=== Update connector pipeline API
+
+preview::[]
+
+++++
+<titleabbrev>Update connector pipeline</titleabbrev>
+++++
+
+Updates the `pipeline` configuration of a connector.
```

Comment:
General question: I think the initial pipeline creation happens automatically as part of connector creation. Do we want to add context like "When you create a connector it automatically adds an ingest pipeline with default settings. Use this API to update the pipeline configuration.". I'm just imagining a new user testing out connectors who doesn't know what an ingest pipeline is...

Reply:
That's a good idea :+1: 

What's the best practice when it comes to version linking between elastic docs and other docs? Should I link to `current` (or latest) or a particular version matching this branch version (but since I want to backport it, this would be tricky)

Reply:
We moved ingest pipelines in Search docs to es docs recently so you wouldn't even need to worry about that. Can just use `<<id>>` syntax. 

Is this the relevant deeplink?

https://www.elastic.co/guide/en/elasticsearch/reference/8.11/ingest-pipeline-search.html#ingest-pipeline-search-details-generic-reference

If yes, then the link syntax would be `<<ingest-pipeline-search-details-generic-reference, LINK TEXT GOES HERE>>`

Reply:
Reminds me we need to update that doc to mention the API option now available!